tests/cmdline/commands/test_code.py

```diff
@@ -236,6 +236,24 @@ def test_code_duplicate_ignore(run_cli_command, aiida_local_code_factory, non_in
     assert duplicate.description == ''
 
 
+def test_code_export(run_cli_command, aiida_local_code_factory, tmp_path, file_regression):
+    """Test export the code setup to str."""
+    code = aiida_local_code_factory('core.arithmetic.add', '/bin/cat', label='code')
+    filepath = tmp_path / 'code.yml'
+    options = [str(code.pk), str(filepath)]
+    run_cli_command(cmd_code.export, options)
+
+    # round trip test by create code from the config file
+    # we pass the new label to override since cannot have two code with same labels
+    new_label = 'code0'
+    run_cli_command(
+        cmd_code.code_create, ['core.code.installed', '--non-interactive', '--config', filepath, '--label', new_label]
+    )
+    new_code = load_code(new_label)
+    assert new_code.use_double_quotes is False
```

Comment:
I take it that `assert code == new_code` won't work since the `AbstractCode` doesn't implement the `__eq__` method. But maybe instead of comparing just this one attribute arbitrarily, we do `assert code.base.attributes.all() == new_code.base.attributes.all()`? That should cover at least all attributes, which for at least the `InstalledCode` should be all relevant data.